filters/mixins.py

```diff
@@ -6,13 +6,12 @@
 from .schema import base_query_params_schema
 
 
-class FiltersMixin(object):
+class FiltersMixin(six.with_metaclass(MetaFiltersMixin, object)):
```

Comment:
can we use @six.add_metaclass decorator instead of six.with_metaclass directly here.
doc says both are equivalent but i felt its cleaner https://pythonhosted.org/six/#six.add_metaclass